CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 All notable changes for each version of this project will be documented in this file.
 ## 11.1.0
+### General
+- `IgxGrid`, `IgxTreeGrid`, `IgxHierarchicalGrid`
+    - The following new events are introduced: `sorting`, `filtering`, `onPaging`, `columnPinned`, `columnVisibilityChanging`.
+    - **Behavioral Change** -
+    - `onColumnPinning` to emit `IPinColumnCancellableEventArgs` instead of `IPinColumnEventArgs`.
```

Comment:
Just these events should no longer be on-prefixed.

Reply:
We should decide whether to rename `paging` property in order to use this name for the event.

Reply:
I have prepared a branch for it (currently adding migrations). When we decide on the name, this branch can be used.

Reply:
#8911 is ready to be tested and merged if `paging` needs to be freed to be used as name for the event.

Reply:
Grid properties related paging should be deprecated. If any input/output handling needs to be done, a template with a paginator should be used and all handling to happen on paginator level.